db/migrate/20170805061539_create_mail_logs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMailLogs < ActiveRecord::Migration
+  def change
+    create_table :mail_logs do |t|
+      t.references :user, index: true
+      t.string :name
+      t.string :email
```

Comment:
I was thinking about storing the user, but will actually remove the user references as a mail log could instead relate to a customer (and possibly other models or none down the line). I haven't decided for sure on sticking to just removing the user, or changing it to a polymorphic relationship so that we can store the reference to the 'user like' object it was delivered to.

But actually storing the name and email here in this case is not bad practice (usually it would be of course) because you're storing the email the mail was sent to - if the user changes their email then you no longer have the actual address the email was sent to.